agent/bench-scripts/test-bin/java

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+""" Emulate / mock the behavior of SPECjbb 2005.
+
+The `pbench-specjbb2005` bench-script runs SPECjbb via the `java` command,
+so we provide this fake `java` executable.
+
+It then reads the generated properties file to figure out how to behave, and
+then sets about created the expected output files in the locations determined
+by the properties.
```

Comment:
Hmm. I could go for `s/then//g` for that matter; I don't think either occurrence is meaningful. If we're tightening the language, I might suggest "and creates the expected output files" rather than "and then sets about creat[ing] the expected output files".

Reply:
Fixing in `main` first ... (see https://github.com/distributed-system-analysis/pbench/pull/2532).